### PR TITLE
Fix MmapAllocator::toString

### PR DESCRIPTION
**Pull request by majetideepak**

### CODE REVIEW

**Comment by majetideepak** [comment]
I will add free capacity value as well for ease of understanding.

File: velox/common/memory/MmapAllocator.cpp [hunk 1]
```diff
@@ -1028,8 +1028,12 @@ bool MmapAllocator::useMalloc(uint64_t bytes) {
...
 std::string MmapAllocator::toString() const {
   std::stringstream out;
-  out << "Memory Allocator[" << kindString(kind_) << " capacity "
-      << ((capacity_ == kMaxMemory) ? "UNLIMITED" : succinctBytes(capacity_))
+  out << "Memory Allocator[" << kindString(kind_) << " total capacity "
+      << ((capacity_ == kMaxMemory) ? "UNLIMITED" : succinctBytes(capacity()))
+      << " free capacity "
+      << ((capacity_ == kMaxMemory)
+              ? "UNLIMITED"
+              : succinctBytes(capacity() - totalUsedBytes()))
```
Comment (xiaoxmeng) [inline velox/common/memory/MmapAllocator.cpp:1036]: use numAllocated()? We don't enforce the malloc memory usage in MmapAllocator. Thanks!

**Review by xiaoxmeng — APPROVED** [review]
@majetideepak LGTM % nit. Thanks!

File: velox/common/memory/MmapAllocator.cpp (hunk 1 above)
Comment (majetideepak) [reply velox/common/memory/MmapAllocator.cpp]: Fixed.

**Pull request closed without merging**